.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug Report
+about: Create a report to help us improve GNU Radio
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# 1. Have you looked in our documentation and asked the community?
+
+<!--- [Before you open an issue we suggest checking the following: -->
+<!--- - [The wiki](https://wiki.gnuradio.org/index.php/Main_Page) -->
+<!--- - [The GNU Radio Blocks documentation](https://wiki.gnuradio.org/index.php/Category:Block_Docs) -->
+<!--- - [The User manual](https://wiki.gnuradio.org/index.php/Usage_Manual) -->
+<!--- - [GNU Radio community](https://wiki.gnuradio.org/index.php/Chat) -->
+<!--- - or on [the mailing list](https://wiki.gnuradio.org/index.php/MailingLists)] -->
+
+# 2. GNU Radio version
+
+
+# 3. How did you install GNU Radio
+<!--- [Choose an option from this list -->
+<!--- - OS package manager -->
+<!--- - Built from source -->
+<!--- - PyBOMBS -->
+<!--- - Anaconda -->
+<!--- - Binary (please link to where you downloaded it) -->
+<!--- - Another way (Describe in 1-2 lines)] -->
+
+# 4. Can you provide a GRC to help us reproduce the issue
```

Comment:
I think this would be better as an subsection of the last item "Additional information"

Reply:
The rationale for asking for a GRC file here rather than in additional information (much later) is - having a GRC would help debugging the related issue significiantly. If we asked for it earlier in the report, there is a chance the user would provide it. Putting it in the last item in "Additional information" the likelihood is they won't.

Maybe we can try it in this position for a while and see how it works? If people complain, then we can ask for it later?

Reply:
Bugs that involve GRC files are kind of a special case; I'm certainly not representative, but the amount of the roundabout 20 issues I've opened this year that involved a GRC file in finding them is actually 0. Of the 25 most recent issues, I think about 3 benefit from having one; GRC is just really one component in many :)

A lot, however, of the users that would very much benefit from this template probably did find the bug they're encountering working in GRC. So, making this visible prominently would be a good thing – although I do agree, would be something on "material".